src/curated-corpus/components/ScheduleItemForm/ScheduleItemForm.tsx

```diff
@@ -113,7 +112,7 @@ export const ScheduleItemForm: React.FC<
 
   useEffect(() => {
     setRefreshData(true);
-    handleDateChange(selectedDate ? selectedDate : tomorrow);
+    handleDateChange(selectedDate);
```

Comment:
Don't set a default date in the form as we already do so in the component above it (ScheduleItemFormConnector), and the form component is never used anywhere directly.